packages/host/app/components/operator-mode/container.gts

```diff
@@ -128,10 +131,7 @@ export default class OperatorModeContainer extends Component<Signature> {
     });
   }
 
-  get stacks() {
-    return this.operatorModeStateService.state?.stacks ?? [];
-  }
-
+  // public API
```

Comment:
redundant comment?

Reply:
I just wanted to make it clear that these are intentionally not private. I see that many times we have a habit of not being clear about what is private and what is not in our components, and that this is not that kind of situation